mm2src/coins/tendermint/rpc/mod.rs

```diff
@@ -0,0 +1,31 @@
+#[cfg(not(target_arch = "wasm32"))] mod tendermint_native_rpc;
+#[cfg(not(target_arch = "wasm32"))]
+pub use tendermint_native_rpc::*;
+
+#[cfg(target_arch = "wasm32")] mod tendermint_wasm_rpc;
+#[cfg(target_arch = "wasm32")] pub use tendermint_wasm_rpc::*;
+
+pub(crate) const TX_SUCCESS_CODE: u32 = 0;
+
+pub enum TendermintResultOrder {
+    Ascending,
+    Descending,
+}
+
+impl From<TendermintResultOrder> for Order {
+    fn from(order: TendermintResultOrder) -> Self {
+        match order {
+            TendermintResultOrder::Ascending => Self::Ascending,
+            TendermintResultOrder::Descending => Self::Descending,
+        }
+    }
+}
+
+impl From<TendermintResultOrder> for i32 {
+    fn from(order: TendermintResultOrder) -> Self {
+        match order {
+            TendermintResultOrder::Ascending => 0,
+            TendermintResultOrder::Descending => 1,
```

Comment:
How about
```rust
#[repr(i32)]
pub enum TendermintResultOrder {
    Ascending = 0,
    Descending,
}
```

Reply:
I saw you added the `repr(u8)` representation. Doesn't `repr(i32)` allow to convert `TendermintResultOrder as i32` free in terms of using registers?

Reply:
No, they both copied to target register. So whatever you cast will be copied to target register. But using `u8` instead of `i32` in enum will allocate less space.

Reply:
Good to know, thanks!